### PR TITLE
Merge pull request #10560 from brave/0_18_23_add_release_notes

### PR DESCRIPTION
**Pull request by bsclifton**
Added release notes for 0.18.x
Fixes #10560

Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).

Test Plan:

Reviewer Checklist:

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+##[0.18.23](https://github.com/brave/browser-laptop/releases/tag/v0.18.23dev)
```
Comment (luixxiul) [inline CHANGELOG.md]: please add a white space after #

**Comment by bsclifton** [comment]
@luixxiul nice catch 😄  updated

**Comment by luixxiul** [comment]
I'm not sure which is the right milestone :-/

**Comment by bsclifton** [comment]
It's OK- we can leave milestone out.  I already merged to other branches- permissions were recently updated for GitHub so I'm unable to merge to master without approval

**Pull request merged**